hazelcast/src/test/java/com/hazelcast/cache/CacheConfigTest.java

```diff
@@ -203,6 +199,19 @@ public void defaultCacheTest() {
     }
 
     @Test
```

Comment:
:+1:  i am definitely not worry :)  only, i want to learn real reason of this style so i can learn new things and apply in my code in the future, @pveentjer